### ISSUE TITLE
boolean union of three meshes

### ISSUE THREAD

**Issue by marcomusy** [marcomusy/vedo#454]
@orgicus
I just received an email notification from stackoverflow but the original message seems to have disappeared so i could not read it, let me know if you need any more help.
https://stackoverflow.com/questions/69145014/how-to-compute-boolean-intersection-between-3-geometries-using-vtk

**Comment by orgicus** [comment]
Hi @marcomusy ,

Thank you for reaching out.

I did temporarily leave a comment about the fact that I've spotted the [underlying VTK calls](https://github.com/marcomusy/vedo/blob/c6adec2092e690d66fd95837907b62dc831b011d/vedo/mesh.py#L1892) for the boolean() operation then felt it was a bit redundant since it didn't provide any information beyond what you already provided in your answer.

I had a quick look at your vedo wrapper it looks great ! Lots of neat examples and the syntax looks so intuitive. 
Good reference/inspiration !
